.swiftpm/xcode/xcuserdata/sujitthorat.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ignore the file changes inside xcuserdata folder. No need to commit these files.